pulp_smash/tests/rpm/api_v2/test_orphan_remove.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+"""Test the removal of rpm orphans.
+
+This module assumes that the tests in
+:mod:`pulp_smash.tests.platform.api_v2.test_repository` and
++`pulp_smash.tests.platform.api_v2.test_sync_publish` hold true.
+"""
+
```

Comment:
Can we add a `from __future__ import unicode_literals` here? It's a nice little way to prevent errors.